libs/ngx-nuts-and-bolts/src/lib/testing/async-data/README.md

```diff
@@ -0,0 +1,23 @@
+# Async data helper
+
+If and when returning mocked data from testing service user should avoid using something like:
+
+```ts
+public getData(data: TData): Observable<TData>{
+	return of(data);
+}
+```
+
+Above example creates observable from the data but it returns it synchronously which isn't how the services that make API calls behave in real world applications.
+The purpose of creating test doubles for services is to mock real world services behavior as much as possible, so with that in mind it is good idea to fake the async behavior of such services, which can be achieved in multiple ways.
```

Comment:
```suggestion
The purpose of creating test doubles for services is to mock the behavior of real world services as closely as possible. With that in mind, it is a good idea to fake the async behavior of such services. This can can be achieved in multiple ways.
```